crypto/evp/evp_enc.c

```diff
@@ -991,10 +996,12 @@ int EVP_CIPHER_CTX_set_key_length(EVP_CIPHER_CTX *c, int keylen)
             return 0;
         }
 
-        params[0] = OSSL_PARAM_construct_size_t(OSSL_CIPHER_PARAM_KEYLEN, &len);
+        params[0] = OSSL_PARAM_construct_int(OSSL_CIPHER_PARAM_KEYLEN, &keylen);
```

Comment:
Not sure I like this auto conversion from an internal  size_t to an int.

Reply:
Changed to size_t.